### DISCUSSION TITLE
通过 Alt + Space 开启内置工具

### DISCUSSION THREAD

**Discussion by imlinhanchao** [he3-app/he3-feedback#40] [Ideas]
通过 Alt + Space 似乎只能唤起系统的软件或文件，不能唤起内置的各种工具，使用工具必须打开软件后在选择使用。如果可以在 Alt + Space 中输入工具名直达工具页，就更加方便了。

**Comment by hongchaodeng** [comment]
非常感谢你的建议！这是一个非常不错的功能。我们记录一下 ❤️
